src/views/AskForHelp.jsx

```diff
@@ -69,10 +69,14 @@ export default function AskForHelp() {
     setPlaceId(pId);
   };
 
-  if (!isAuthLoading && (!user || !user.email)) {
+  if (!user || (!isAuthLoading && (!user || !user.email))) {
```

Comment:
The latter !user will always be false so I guess the expression can be refined
```suggestion
  if (!user || (!isAuthLoading && !user.email)) {
```